tests/tests_basic_join.yml

```diff
@@ -1,10 +1,37 @@
 # SPDX-License-Identifier: MIT
 ---
-- name: Ensure that the role runs with default parameters
-  hosts: all
+- name: Ensure that the role runs with manadatory parameters populated
+  hosts: all,!ad
   gather_facts: false  # test that role works in this case
-  roles:
-    - linux-system-roles.template
   vars:
-    ad_integration_realm: domain.com
+    ad_integration_realm: "{{ __ad_integration_dummy_realm }}" # dummy realm that will skip joining realm step
     ad_integration_password: Secret123
+
+  tasks:
+    - name: Run the system role
+      include_role:
+        name: linux-system-roles.template
+
+    - name: Test - check packages are present
+      block:
+      - name: Gather package facts
+        package_facts:
+          manager: auto
```

Comment:
`manager: auto` is the default - you can omit this line

Reply:
Resolved.